vendor/engines/secure_rooms/spec/services/secure_rooms/access_handlers/order_handler_spec.rb

```diff
@@ -58,6 +59,34 @@
           expect(order_detail.created_by_user).to eq user
         end
       end
+
+      context "when completing the occupancy" do
+        let(:occupancy) do
+          create(
+            :occupancy,
+            :complete,
+            user: user,
+            secure_room: secure_room,
+            account: account,
+          )
+        end
+
+        describe "order_detail" do
+          subject(:order_detail) { described_class.process(occupancy).order_details.first }
+
+          it { is_expected.to be_complete }
+          it { is_expected.to be_fulfilled_at }
+
+          # TODO: only correct until the subsequent price specs are activated
+          it { is_expected.to be_problem }
+
+          xit "has price information" do
+            # TODO: expand once activated
+            expect(order_detail.price_policy).to be_present
```

Comment:
I just merged in #1001, which should have the calculations. I think if you rebase on master, you can probably un-pend this.